Library/.rubocop_rules.yml

```diff
@@ -1,75 +1,197 @@
-# ruby style guide favorite
-Style/StringLiterals:
-  EnforcedStyle: double_quotes
-
-# consistency with above
-Style/StringLiteralsInInterpolation:
-  EnforcedStyle: double_quotes
-
-# only for numbers >= 1_000_000
-Style/NumericLiterals:
-  MinDigits: 7
-
-# zero-prefixed octal literals are just too widely used (and mostly understood)
-Style/NumericLiteralPrefix:
-  EnforcedOctalStyle: zero_only
-
-# percent-x is allowed for multiline
-Style/CommandLiteral:
-  EnforcedStyle: mixed
-
-# depends_on foo: :bar looks rubbish
-Style/HashSyntax:
-  EnforcedStyle: ruby19
-  Exclude:
-    - 'Taps/**/*'
-
-# paths abound, easy escape
-Style/RegexpLiteral:
-  EnforcedStyle: slashes
-
-Style/Alias:
-  EnforcedStyle: prefer_alias
-
-# our current conditional style is established, clear and
-# requiring users to change that now would be confusing.
-Style/ConditionalAssignment:
-  Enabled: false
-
-# no metrics for formulas
 Metrics/AbcSize:
   Enabled: false
-Metrics/CyclomaticComplexity:
-  Enabled: false
-Metrics/MethodLength:
-  Enabled: false
+
 Metrics/ClassLength:
   Enabled: false
-Metrics/PerceivedComplexity:
+
+Metrics/CyclomaticComplexity:
   Enabled: false
 
-# we often need very long lines
 Metrics/LineLength:
   Enabled: false
 
-# formulas have no mandatory doc
-Style/Documentation:
+Metrics/MethodLength:
+  Enabled: false
+
+Metrics/ModuleLength:
+  CountComments: false
+  Exclude:
+    - 'Homebrew/cask/lib/hbc/locations.rb'
+    - 'Homebrew/cask/lib/hbc/macos.rb'
+    - 'Homebrew/cask/lib/hbc/utils.rb'
+
+Metrics/PerceivedComplexity:
   Enabled: false
 
 # favor parens-less DSL-style arguments
 Lint/AmbiguousOperator:
   Enabled: false
+
 Lint/AmbiguousRegexpLiteral:
   Enabled: false
+
 Lint/AssignmentInCondition:
   Enabled: false
+
+Lint/EndAlignment:
+  AlignWith: variable
+
+# `formula do` uses nested method definitions
+Lint/NestedMethodDefinition:
+  Exclude:
+    - 'Homebrew/test/**/*'
+
 Lint/ParenthesesAsGroupedExpression:
   Enabled: false
 
-# compact style
+Style/Alias:
+  EnforcedStyle: prefer_alias
+
+Style/AlignHash:
```

Comment:
I reckon it's worth not having this because of the realignment required when different length keys are added to a large existing hash and the expanding diff that results.